master/painting.html

```diff
@@ -1222,14 +1224,45 @@ <h3 id="StrokeDashing">Dashing strokes: the <span class="property">'stroke-dasha
 <a>'stroke-dasharray'</a>: each dash and gap length is interpreted relative
 to the author's path length as specified by <a>'path/pathLength'</a>.</p>
 
+<p id="interpolationDashPattern"><a>'stroke-dasharray'</a> must use discrete
+interpolation if <var>start</var> or <var>end</var> value compute to <span class="prop-value">none</span>
+or are invalid.
+Otherwise, repeat both dash patterns of <var>start</var> and <var>end</var> value list until the length
+of elements in both value lists match. Each item is then combined by computed value.
+The interpolation is <a href="https://svgwg.org/specs/animations/#AdditionAttributes">non-additive</a>.</p>
```

Comment:
We should link to CSS V&U 4 here: https://drafts.csswg.org/css-values-4/#not-additive

Reply:
done (even though it is ED :)). Added it to the linking section in definitions.xml for easy edit.